raster/r.mfilter/main.c

```diff
@@ -132,10 +132,11 @@ int main(int argc, char **argv)
                     "threads setting."));
     nprocs = 1;
 #endif
-    if (nprocs > 1 && G_find_raster("MASK", G_mapset()) != NULL) {
-        G_warning(_("Parallel processing disabled due to active MASK."));
-        nprocs = 1;
-    }
+    if (nprocs > 1 && Rast_mask_is_present()) != NULL)
+        {
+            G_warning(_("Parallel processing disabled due to active mask."));
+            nprocs = 1;
+        }
```

Comment:
I get errors in the CI for Clang-format and locally it does this (curly brackets indented), so I'm both confused and unable to easily fix it right now.

Reply:
There's an active incident for the docker hub registry, where only anonymous requests are working. Authenticated (like on github actions here) are failing, so pulling some images that are used in CI aren't working

Reply:
Well, no the behavior of CI is consistent with what I get locally, but it's wrong, no? We have opening `{` on the same line and not two different indents for `{}` and the code. I see there is a new major version of https://github.com/pre-commit/mirrors-clang-format but the version in our pre-commit is 18.

Reply:
Oh, I see. I had a syntax error just above that which I ignored, and the somewhat reasonable, but wrong formatting got me on the wrong track.